submissions/damilolaSalami/css/signUp.css

```diff
@@ -0,0 +1,416 @@
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+.body{
+    font-family: Arial, Helvetica, sans-serif;
+    }
+
+/* header */
+.header{
+    background-color: purple;
+    align-items: center;
+    justify-content: space-between;
+    display: flex;
+    height: 75px;
+    padding-top: 0;
+}
+
+.header .logo img{
+    height: 75px;
```

Comment:
You should set only either of the height or width. That way, the image can resize based on screen size rather than having it static @queensaldam

Reply:
You should set only either of the height or width. That way, the image can resize based on screen size rather than having it static @queensaldam